less/hotgrid.less

```diff
@@ -117,7 +122,7 @@
 				width: 100%;
 				background-color: fade(@item-color-disabled, 50%);
 				/* uncomment if circular items */
-				//border-radius: 50%; 
+				border-radius: 50%; 
```

Comment:
should this line be included? isn't it now handled by lines 38 - 41? either way the comment should be removed

Reply:
Yep it's not needed now, will delete